### PR TITLE
Adding value parameter to variable variable_header_format_default

### PR DESCRIPTION
**Pull request by dl1ksv**
Adding this block to a flowgraph produces error messages. 
See #2619 
This proposal comes accross this problem

### CODE REVIEW

File: gr-digital/grc/variable_header_format_default.block.yml [hunk 1]
```diff
@@ -15,12 +15,17 @@ parameters:
...
+    dtype: string
+    default: 'digital.header_format_default(access_code, threshold, bps)'
+    hide: 'all'
+value: ${ value }
 
 templates:
     imports: from gnuradio import digital
     var_make: |-
-        % if int(eval(access_code))==0:
+        % if access_code == '0':
```
Comment (jdemel) [inline gr-digital/grc/variable_header_format_default.block.yml]: What happens if one puts `0000`? Should this result in an all zero access code? Or should this also trigger the default access code? At least this does change the blocks behavior.

File: gr-digital/grc/variable_header_format_default.block.yml [hunk 2]
```diff
@@ -32,7 +37,7 @@ cpp_templates:
     includes: ['#include <gnuradio/digital/header_format_default.h>']
     declarations: 'digital::header_format_default::sptr ${id};'
     var_make: |-
-        % if int(eval(access_code))==0:
+        % if access_code == '0' :
```
Comment (jdemel) [inline gr-digital/grc/variable_header_format_default.block.yml]: See line 28 comment.

File: gr-digital/grc/variable_header_format_default.block.yml [hunk 3]
```diff
@@ -15,12 +15,17 @@ parameters:
     label: Payload Bits per Symbol
     dtype: int
     default: 1
-value: ${ digital.header_format_default(access_code, threshold, bps) }
```
Comment (jdemel) [inline gr-digital/grc/variable_header_format_default.block.yml]: I assume this is the actual change, that should fix this issue?

**Comment by dl1ksv** [comment]
But as there exist about 10 block that own the same 
KeyError: 'value' problem, I think this problem should be fixed inside grc and not by modifying the yml description.
I'm searching for a solution, but I've not finished yet. You can find some thougths at the end of #2605

**Comment by dl1ksv** [comment]
This issue should better be fixed inside grc. See #2662

**Pull request closed without merging**